scripts/process_data.py

```diff
@@ -732,7 +296,17 @@ def main(self) -> None:
 
 
 @dataclass
-class ProcessRealityCapture:
+class _NoDefaultrocessRealityCapture:
```

Comment:
`_NoDefaultrocessRealityCapture` -> `_NoDefaultProcessRealityCapture:`

Reply:
Thanks for the comment, it's corrected.